src/test/testdata/lsl/shiftSquareBrackets.lsl

```diff
@@ -0,0 +1,31 @@
+libsl "1.0.0";
+library simple;
+
+types {
+    int(int32);
+}
+
+automaton a: int {
+
+    state S1;
+    state S2;
+    state S3;
+
+    shift S1 -> S2 by foo;
+    shift S1 -> S2(fix);
+    shift S1 -> S2 by [fix, foo,];
+    shift S1 -> S2 by [foo];
+    shift S1 -> S2 by [foo,];
+    shift S1 -> S3 by [sum(int, int), min(int), ];
+    shift S1 -> S3 by sum(int, int);
+
+    shift (S1, S2) -> S3 by foo;
+    shift (S1, S2) -> S3 by [sum(int, int), min(int),];
```

Comment:
Could you add at least one test with the new syntax (with brackets) with 2+ args and without trailing comma?

Or you could remove the comma from this line

Reply:
Done!